sdks/java/core/src/main/java/org/apache/beam/sdk/coders/SerializableCoder.java

```diff
@@ -224,6 +224,11 @@ public int hashCode() {
     return typeDescriptor;
   }
 
+  @Override
+  public String toString() {
```

Comment:
This should be more of a debug string where no info is skipped, like `MoreObjects.toStringHelper(this).add(clazz)`

Reply:
I've added the type here